catalog/app/components/JsonEditor/Cell.tsx

```diff
@@ -86,6 +86,8 @@ export default function Cell({
 
   const onKeyPress = React.useCallback(
     (event) => {
+      if (editing) return
+
       if (!editing) {
```

Comment:
given previous statement, the condition will be always true, so `if` isnt required anymore